### PR TITLE
Remove sorting for election results view

### LINKED ISSUE #36: Remove sorting from summary table

### PR DESCRIPTION
**Pull request by dedavidson**

### CODE REVIEW

File: localgov_elections_reporting.install [hunk 1]
```diff
@@ -174,4 +174,27 @@ function localgov_elections_reporting_update_9003(&$sandbox){
   $charts_config = \Drupal::configFactory()->getEditable('charts.settings');
   $charts_config->set('advanced.requirements.cdn', TRUE);
   $charts_config->save();
-}
\ No newline at end of file
+}
+
```
Comment (chriswales95) [inline localgov_elections_reporting.install:178]: I'd add a comment here so it tells people what the update does. When you do `drush updb` the table will tell the user what the update is going to do.

**Review by chriswales95 — CHANGES_REQUESTED** [review]

File: localgov_elections_reporting.install [hunk 2]
```diff
@@ -174,4 +174,27 @@ function localgov_elections_reporting_update_9003(&$sandbox){
...
+        $data = $install_dir->read($config);
+        $config_obj = \Drupal::configFactory()->getEditable($config);
+        $config_obj->setData($data);
+        $config_obj->save(TRUE);
+    }
+
+    Drupal::logger('localgov_elections_reporting')->notice("Updated election results view to remove sorting");
+
+    drupal_flush_all_caches();
+}
```
Comment (chriswales95) [inline localgov_elections_reporting.install]: I'd add a comment to this install hook so it tells the user what the update does when they run `drush updb`

**Review by chriswales95 — CHANGES_REQUESTED** [review]

File: localgov_elections_reporting.install (hunk 2 above)
Comment (dedavidson) [reply localgov_elections_reporting.install]: Done

**Review by chriswales95 — APPROVED** [review]

**Comment by chriswales95** [comment]
@dedavidson Looks good to me!

**Pull request merged**